experimental/QuadFormAndIsom/test/setup_tests.jl

```diff
@@ -1,41 +0,0 @@
-# This code is needed for multiple test files that may end up on different workers.
```

Comment:
There was some reason why this wasn't set up as a symlink, but I unfortunately cannot find it right now. It would not change that in this PR at least

Reply:
Sorry, stupid comment